client/packages/host/src/Host.tsx

```diff
@@ -108,28 +111,28 @@ const router = createBrowserRouter(
 const Host = () => (
   <React.Suspense fallback={<div />}>
     <IntlProvider>
-      <React.Suspense fallback={<RandomLoader />}>
-        <ErrorBoundary Fallback={GenericErrorFallback}>
-          <QueryClientProvider client={queryClient}>
-            <GqlProvider
-              url={Environment.GRAPHQL_URL}
-              skipRequest={skipRequest}
-            >
-              <AuthProvider>
-                <Init />
-                <AppThemeProvider>
+      <AppThemeProvider>
```

Comment:
moving up app theme, so both error boundaries both have the theme applied (for same coloured buttons)